src/holochain/app_conn_pool.rs

```diff
@@ -32,15 +33,17 @@ pub struct AppWebsocketWithState {
 #[derive(Debug, Clone)]
 pub struct AppConnPool {
     configuration: Configuration,
+    admin_call: Arc<dyn AdminCall>,
```

Comment:
Can hold a reference to this because it reconnects. That wasn't the case when using an `AdminWebsocket` directly.